modules/measurement/budget-calculator/load.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * Module Name: Budget Calculator
+ * Description: Allows users to define their own metrics and generate a budget.json file from them.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Display the content of the budget calculator option page.
+ *
+ * @since n.e.x.t
+ */
+function budget_calc_page_content() {
+	?>
+	<div class="wrap">
+	<h1>Budget Calculator</h1>
```

Comment:
Please use the escape function.

`esc_html_e( 'Budget Calculator', 'performance-lab' );`